README.md

```diff
@@ -12,7 +12,7 @@ See page: https://menghanl.github.io/release-git-bot/
 ### Install or update the tool:
 
 ```
-go get -u github.com/menghanl/release-git-bot
+GO111MODULE=on go install github.com/menghanl/release-git-bot
```

Comment:
Can you also update this line: https://github.com/menghanl/release-git-bot/blob/972c54a3627b434d09f994d0d0e028d58b2c3392/docs/index.html#L35?

Thanks!

Reply:
Done.